### PR TITLE
Refresh contributors file for the 1.20 release

### PR DESCRIPTION
**Pull request by bradcray**

### CODE REVIEW

**Comment by bradcray** [comment]
@chapel-lang/core-contributors: Would you check to see that I've correctly included any external contributors who you worked with during this release?

@dgarvit, @krishnadey30, @mnafees, @wongalvis14: Would you check that I've spelled your names correctly and gotten your schools correct?

@akshansh2000, @sanket143, @hackerde, @dtjong, @AnubhavUjjawal: Would you check that I've spelled your names correctly?

Thanks!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -7,26 +7,33 @@ Chapel release:
...
-* Sidhant Bendre, individual contributor
+* Akshansh Bhanjana, individual contributor
 * Paul Cassella, [Cray Inc.]
 * [Brad Chamberlain], [Cray Inc.]
-* Naman Chikara, individual contributor
-* Krishna Kumar Dey, individual contributor
+* Sanket Chaudhari, individual contributor
+* Anway De, individual contributor
+* Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology, Roorkee]
+* Krishna Kumar Dey, [GSoC 2019] student from [Indian Institute of Information Technology, SriCity] (former individual contributor)
```
Comment (krishnadey30) [inline CONTRIBUTORS.md]: Can you correct the name of Institute to `Indian Institute of Information Technology, Sri City`.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -182,8 +192,11 @@ implementation.
 [GSoC 2016]: https://summerofcode.withgoogle.com/organizations/6608321708556288/
 [GSoC 2017]: https://summerofcode.withgoogle.com/organizations/6472441615351808/
 [GSoC 2018]: https://summerofcode.withgoogle.com/organizations/4743716460298240/
+[GSoC 2019]: https://summerofcode.withgoogle.com/organizations/4892546359099392/
 [Haverford College]: https://www.haverford.edu/
+[Indian Institute of Information Technology, SriCity]: http://www.iiits.ac.in/
```
Comment (krishnadey30) [inline CONTRIBUTORS.md]: Update to `Indian Institute of Information Technology, Sri City`.

**Comment by lydia-duncan** [comment]
All the contributors I have worked with this release are listed

File: CONTRIBUTORS.md (hunk 1 above)
Comment (bradcray) [reply CONTRIBUTORS.md]: Yes, thanks for catching that.  I took it from their LinkedIn page, thought it seemed odd, and then failed to notice that their actual webpage had the space.

Comment (krishnadey30) [reply CONTRIBUTORS.md]: Thanks. I need to update on my LinkedIn page as well.

**Comment by mppf** [comment]
@bradcray - I think we should include Himanshu Bhadani @himbhadani for PR #12625.

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -7,26 +7,33 @@ Chapel release:
...
 * Ben Albrecht, [Cray Inc.]
-* Sidhant Bendre, individual contributor
+* Akshansh Bhanjana, individual contributor
 * Paul Cassella, [Cray Inc.]
 * [Brad Chamberlain], [Cray Inc.]
-* Naman Chikara, individual contributor
-* Krishna Kumar Dey, individual contributor
+* Sanket Chaudhari, individual contributor
+* Anway De, individual contributor
+* Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology, Roorkee]
```
Comment (dgarvit) [inline CONTRIBUTORS.md]: Hi, can you remove the `,` between Technology and Roorkee? The name of my school is Indian Institute of Technology Roorkee (without the `,`).

**Comment by bradcray** [comment]
Thanks for catching that @mppf...  it seems I didn't go quite far enough back in the date range (I think I forgot how much earlier the spring release is now).

File: CONTRIBUTORS.md (hunk 3 above)
Comment (bradcray) [reply CONTRIBUTORS.md]: Yep, no problem.

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -7,26 +7,34 @@ Chapel release:
...
-* Naman Chikara, individual contributor
-* Krishna Kumar Dey, individual contributor
+* Sanket Chaudhari, individual contributor
+* Anway De, individual contributor
+* Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology Roorkee]
+* Krishna Kumar Dey, [GSoC 2019] student from [Indian Institute of Information Technology, Sri City] (former individual contributor)
 * Lydia Duncan, [Cray Inc.]
 * Michael Ferguson, [Cray Inc.]
 * Ben Harshbarger, [Cray Inc.]
 * Jenna Hoole, [Cray Inc.]
```
Comment (jhoole) [inline CONTRIBUTORS.md]: I am "Jenna Hoole Starkey" (or just "Jenna Starkey" if we don't like middle names) these days.

**Comment by mppf** [comment]
@bradcray - should we include Karlon because of PR #12746 ?

**Comment by bradcray** [comment]
@mppf: Sure.  Strangely, that PR's commits don't seem to show up for me in this view which is how I missed it:  https://github.com/chapel-lang/chapel/graphs/contributors?from=2019-03-11&to=2019-09-03&type=c

**Pull request merged**